.github/workflows/percy-prepare.yml

```diff
@@ -0,0 +1,48 @@
+name: "Prepare Percy build"
```

Comment:
Is it a workflow, or an action?

It doesn't seem to be triggered via PR events, but reused in workflows as if it was an action, right?

Reply:
It would be good to have a comment on top what the workflow does, and when it's supposed to be triggered, to have a better understanding of the flow. 

As now it's getting complicated with multiple conditional workflows.

Reply:
@bartaz 
> Is it a workflow, or an action?
It is a [reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows). 

> It doesn't seem to be triggered via PR events, but reused in workflows as if it was an action, right?
Yes, it's triggered by the snapshot & label workflows - the workflow runs at job level whereas the action runs at the step level

Reply:
> It would be good to have a comment on top what the workflow does, and when it's supposed to be triggered, to have a better understanding of the flow.
> 
> As now it's getting complicated with multiple conditional workflows.

Agreed, I'll add a note to the workflow momentarily